csrankings-5.csv

```diff
@@ -2179,8 +2179,8 @@ Michael Jenkin,York University,http://eecs.lassonde.yorku.ca/faculty/michael-r-m
 Michael Johnson,Macquarie University,http://web.science.mq.edu.au/~mike,NOSCHOLARPAGE
 Michael Julian Black [IS],Max Planck Society,https://ps.is.tuebingen.mpg.de/person/black,6NjbexEAAAAJ
 Michael Jurczyk,University of Missouri,https://engineering.missouri.edu/faculty/michael-jurczyk,NOSCHOLARPAGE
-Michael K. Ng,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
-Michael K. P. Ng,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~mng,BBpjLiIAAAAJ
+Michael K. Ng,University of Hong Kong,https://hkumath.hku.hk/~mng/,BBpjLiIAAAAJ
+Michael K. P. Ng,University of Hong Kong,https://hkumath.hku.hk/~mng/,BBpjLiIAAAAJ

```

Comment:
Not in CS, not listed here: https://www.cs.hku.hk/people/academic-staff.

Reply:
This is to update affiliation of existing entry. I can remove it if you want.